pkg/virt-api/webhooks/utils.go

```diff
@@ -77,3 +77,11 @@ type Informers struct {
 func IsARM64(vmiSpec *v1.VirtualMachineInstanceSpec) bool {
 	return vmiSpec.Architecture == "arm64"
```

Comment:
To cover heterogenous clusters, please also decide based on Default VM architecture when it is unset.

Same for the other arches, so maybe make that one a small utils function:
```
func IsDefaultArch(vmiSpec *v1.VirtualMachineInstanceSpec, arch string, , config *virtconfig.ClusterConfig) bool {
    return  vmiSpec.Architecture == "" && config.ArchitectureConfiguration.DefaultArchitecture == arch
}
```